db/seeds/dummy_projects.rb

```diff
@@ -48,7 +48,6 @@ def users
     user = User.new(user_details)
     user.password = "password"
     user.password_confirmation = "password"
-    user.skip_confirmation!
```

Comment:
Why was this removed?

We could also add `user.confirm` to make sure the user account is confirmed.